Modules/ASEP/Get-PersistanceFilesAndRegistryKeys.ps1

```diff
@@ -0,0 +1,338 @@
+﻿<#
+.SYNOPSIS
+Get-SvcAll.ps1 returns information about all Windows services.
+
+.NOTES
+The following line is required by Kansa.ps1, which uses it to determine
+how to handle the output from this script.
+OUTPUT CSV
```

Comment:
Output directives are obsolete, these lines can be taken out. Kansa will default to CSV output and this can be controlled via the OutputFormat parameter.